desktop/keepintouch-100.php

```diff
@@ -0,0 +1,318 @@
+      background: url('/cdn/dev/img/keyman-48.png') 8px 0 no-repeat;
+      font: bold 24pt Cabin;
+    }
+    
+    h2 { 
+      float: left;
+      clear: left;
+      width: 140px;
+      margin: 0 30px 20px 0;
+      font: 22pt Cabin;
+    }
+    
+    #mailchimp h2 {
+      width: 250px;
+    }
+    
+    p, li {
+      font: 12pt Cabin;
+    }
+    
+    .container {
+      float: left;
+    }
+    
+    #facebook .container {
+      margin-top: 8px;
+    }
+    
+    #twitter .container {
+      margin-top: 2px;
+    }
+    
+    #google-plus .container {
+      margin-top: 6px;
+    }
+    
+    #mailchimp .container {
+      margin-top: 4px;
+    }
+    
+    .mc-field-group, .mc-submit {
+      float: left;
+    }
+
+    #mailchimp input {
+      font: 14pt Cabin;
+      height: 30px;
+      box-sizing: border-box;
+    }
+    
+    #ribbon { 
+      position: absolute; 
+      left: 0; 
+      top: 0; 
+      width: 100%; 
+      height: 8px; 
+      background: #79C3DA; 
+    }
+
+    #ribbon div {
+      position: static;
+      float: left;
+      height: 8px;
+    }
+
+    #ribbon #ribbon1 {
+      width: 56%;
+      background: #F68924;
+    }
+
+    #ribbon #ribbon2 {
+      width: 24%;
+      background: #CC3846;
+    }
+    
+    .twitter-follow-button {
+      background: url('/cdn/dev/img/twitter-keep-in-touch.png');
+      width: 144px;
+      height: 28px;
+      display: inline-block;
+    }
+    
+  </style>
+</head>
+<body>
+<div class='embed' id="ribbon">
+  <div id="ribbon1"></div>
+  <div id="ribbon2"></div>
+  <div id="ribbon3"></div>
+</div>
+
+<h1>Keep in touch with the Keyman team</h1>
+
+<div class='content'>
+  <ul>
+    <li>Communicate directly with the people who created Keyman
+    <li>Learn tips and tricks for using your language more effectively on your computer
+    <li>Get news of updates, features and special offers
+  </ul>
+
+  <p class='embed browser'>You can also <a href='.' target='_blank'>open this page in your browser</a></p>
+
+  <div id='facebook'>
+    <h2>Facebook</h2>
+
+    <div class='container'>
+      <div id="fb-root"></div>
+      <div class="fb-like" data-href="https://facebook.com/KeymanApp" data-width="200" data-layout="button_count" data-action="like" data-show-faces="false" data-share="false"></div>
+    </div>
+  </div>
+
+  <div id='twitter'>
+    <h2>Twitter</h2>
+
+    <div class='container'>
+      <a href="https://twitter.com/keyman" class="twitter-follow-button" data-show-count="false" data-size="large" data-dnt="true" target='_blank'></a>
+    </div>
+  </div>
+
+  <div id='mailchimp'>
+    <h2>Keyman Mailing List</h2>
+
+    <div class='container'>
+      <!-- Begin MailChimp Signup Form -->
+      <div id="mc_embed_signup">
+      <form action="https://tavultesoft.us1.list-manage.com/subscribe/post-json?u=99fcab2b035a8a51cd2158ca9&amp;id=7ccdac1e32&amp;c=?" method="get" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" class="validate"  novalidate>
```

Comment:
Just double-checking the link is correct?

Reply:
Yes it is correct. Verified.